startup.d/10-git-pull.sh

```diff
@@ -1,2 +1,21 @@
 #!/bin/sh
+
+# Prepare code auditing environment
+REPO_URL=$(git remote get-url origin)
+TEST_DIR=$(mktemp -d)
+git clone "$REPO_URL" "$TEST_DIR";
```

Comment:
https://stackoverflow.com/a/1210012/345059 could be used to speed up the temp clone as the repo grows